src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age = null,
+    sex,
```

Comment:
I guess there is no need to assign any default value to age property. Later, the logical operator && will do all the work for us.

Reply:
done